### PR TITLE
chore: file methods for scorecards in NFTFeed and NAVFeed

### PR DESCRIPTION
**Pull request by manuelpolzhofer**
- added file method to NFT Feed for score cards
   - no scorecard by default in NFT Feed
- added file method to NFT Feed for score card with additional recoveryRatePD
- default score card for testing is only initialised in NAV Feed init method

### CODE REVIEW

File: src/borrower/feed/navfeed.sol [hunk 1]
```diff
@@ -56,24 +56,68 @@ contract NAVFeed is BaseNFTFeed, Interest, Buckets, FixedPoint {
     // This is required for more accurate Senior & JuniorAssetValue estimations between epochs
     uint public approximatedNAV;
 
-    uint constant public  WRITE_OFF_PHASE_A = 91;
-    uint constant public  WRITE_OFF_PHASE_B = 90;
+    // rate group for write-offs in pile contract
+    uint constant public  WRITE_OFF_PHASE_A = 1001;
```
Comment (charleenfei) [inline src/borrower/feed/navfeed.sol:60]: is this update to the write off phase correct?

**Review by charleenfei — APPROVED** [review]

File: src/borrower/feed/nftfeed.sol [hunk 2]
```diff
@@ -128,11 +90,14 @@ contract BaseNFTFeed is DSNote, Auth, Math {
...
-        // set interestRate for risk group
-        pile.file("rate", risk_, rate_);
+    function file(bytes32 name, uint risk_, uint thresholdRatio_, uint ceilingRatio_, uint rate_) public auth {
+        if(name == "riskGroupNFT") {
+            require(ceilingRatio[risk_] == 0, "risk-group-in-usage");
+            thresholdRatio[risk_] = thresholdRatio_;
+            ceilingRatio[risk_] = ceilingRatio_;
+            // set interestRate for risk group
+            pile.file("rate", risk_, rate_);
+        } else {revert ("unkown name");}
```
Comment (charleenfei) [inline src/borrower/feed/nftfeed.sol:100]: typo

File: src/borrower/feed/navfeed.sol [hunk 3]
```diff
@@ -82,6 +126,14 @@ contract NAVFeed is BaseNFTFeed, Interest, Buckets, FixedPoint {
         setWriteOff(1, WRITE_OFF_PHASE_B, uint(1000000674400000000000000000), 8 * 10**26);
     }
 
+    function file(bytes32 name, uint risk_, uint thresholdRatio_, uint ceilingRatio_, uint rate_, uint recoveryRatePD_) public auth  {
+        if(name == "riskGroup") {
+            file("riskGroupNFT", risk_, thresholdRatio_, ceilingRatio_, rate_);
+            recoveryRatePD[risk_] = Fixed27(recoveryRatePD_);
+
+        } else {revert ("unkown name");}
```
Comment (charleenfei) [inline src/borrower/feed/navfeed.sol:134]: typo

**Review by ilinzweilin — APPROVED** [review]

File: src/borrower/feed/navfeed.sol (hunk 3 above)
Comment (manuelpolzhofer) [inline src/borrower/feed/navfeed.sol:134]:
```suggestion
        } else {revert ("unknown name");}
```

**Pull request merged**